CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 =========
 
+Version 0.1.4
+=============
+- Updates to documentation
+- Adding extra function for loading rainfall files ```load_rain_file_flanders```
+- Code refactoring
+- Adding unit tests
```

Comment:
Can be removed, these are the unit tests that are part of the new function load_rain_file_flanders.